### DISCUSSION TITLE
Improvement of Comments Feature - Screenshots and Screencasts

### DISCUSSION THREAD

**Discussion by MAN-Sendance** [vercel/community#2880] [Idea]
It would be really helpful if
- the screenshots feature would be more sophisticated (see netlify)
- besides screenshots also screencasts would be possible (see netlify)
- both screenshots and screencasts would be added as comments to pull request/merge requests

**Comment by smaeda-ks** [comment]
Thanks for the feedback. I've passed this on to the team internally.
